src/components/MessagesList/MessagesList.vue

```diff
@@ -1215,4 +1380,13 @@ export default {
 		margin-bottom: 16px;
 	}
 }
+
+.has-sticky .messages-group__date {
+	transition: opacity 0.3s ease-in-out;
+	opacity: 0;
+}
+
+.scroller--isScrolling .has-sticky .messages-group__date {
+  opacity: 1;
```

Comment:
```suggestion
	transition: opacity 0s;
	opacity: 1;
```
To show date immediately